app/components/container/form-volumes/component.js

```diff
@@ -265,8 +265,9 @@ export default Component.extend({
         };
       } else if ( volume.flexVolume && volume.flexVolume.driver === LOG_AGGREGATOR ) {
         entry = {
-          mode: C.VOLUME_TYPES.CUSTOM_LOG_PATH,
+          mode:   C.VOLUME_TYPES.CUSTOM_LOG_PATH,
```

Comment:
Purely stylistic but can you move the single property assignments to the top of objects. IMO it looks way cleaner and easier to read.